src/main/java/com/rybka/service/CSVExportService.java

```diff
@@ -0,0 +1,24 @@
+package com.rybka.service;
+
+import com.opencsv.CSVWriter;
+import com.rybka.model.CurrencyHistory;
+import lombok.extern.log4j.Log4j;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+@Log4j
+public class CSVExportService {
+
+    public void export(List<CurrencyHistory> histories, String file) {
+        try (var fileWriter = new FileWriter(file); var csvWriter = new CSVWriter(fileWriter)) {
```

Comment:
I believe CSVMapper(from jackson) is able to map and write list of data to the file. Check that please.

Reply:
fixed